e2e-test/config.go

```diff
@@ -38,14 +38,14 @@ func NewDefaultConfig() *Config {
 		ChartDir:             "/charts",
 		ManifestDir:          "/manifests",
 		Tag:                  "e2e",
-		ManagerImageRegistry: "localhost:5000",
```

Comment:
If you'd like to modify these files, please submit another PR.

Reply:
It seems you are reviewing an outdated commit, please refresh and continue.